### PR TITLE
fix: remove abs for cashflow

### PR DESCRIPTION
**Pull request by ssbarbee**

### CODE REVIEW

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const CurrentMargin: React.FunctionComponent<CurrentMarginProps> = ({
       return 'Margin';
     }
 
-    const formattedAccFlow = formatCurrency(Math.abs(accruedCashflow));
```
Comment (0xZenus) [inline src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:46]: We need to undo this change and check against ``accruedCashflow`` in here ``{accFlowValue > 0 ? '+' : accFlowValue < 0 ? '-' : ''}`` (instead of ``accFlowValue``).

**Review by 0xZenus — CHANGES_REQUESTED** [review]

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx (hunk 1 above)
Comment (ssbarbee) [reply src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:46]: fixed in c6182399cab3cc2cd6d3f4eb3a493b035d3a9344 have a look

**Review by 0xZenus — APPROVED** [review]
